docbook/src/docbkx/autoscale-devguide.xml

```diff
@@ -2201,6 +2229,20 @@ config at https://github.com/rackerlabs/autoscaling-chef/blob/master/site-cookbo
                 server is, and which cloud load balancer or
                 RackConnectV3 load balancer pool the new server
                 connects to.</para>
+            <note>
+                <itemizedlist>
+                    <listitem><para>
+                        The <parameter>launchConfiguration</parameter> uses the <emphasis role="bold">admin</emphasis> user’s SSH key pair name, 
```

Comment:
A previous comment was changed (~line 551), this one says could also use the same change though.  :)